tests/python/test_lang.py

```diff
@@ -131,3 +132,29 @@ def test():
         print(i)
 
     test()
+
+
+@ti.test(require=ti.extension.data64)
```

Comment:
IIUC, this applies to all the test cases, so platforms like Metal cannot be covered, even for 32-bit types? If so, could you split this into two cases, one for 64-bit and the other for non-64?